src/main/steps/applicant1/hub-page/joint/content.ts

```diff
@@ -192,6 +192,11 @@ const cy: typeof en = ({ isDivorce, userCase, partner, isApplicant2 }: CommonCon
       isDivorce ? 'broses ysgaru' : "broses i ddod â'ch partneriaeth sifil i ben"
     }.`,
   },
+  finalOrderComplete: {
+    line1: 'Your final order has been granted. You are now legally divorced.',
+    line2: `<a class="govuk-link" href="/downloads/final-order-granted" download="Final-Order-Granted">Download a copy of your ‘final order’</a>.
```

Comment:
we normally split up the html stuff so the tags are in the html and the content is broken up. if you search part1 youll see how its split like
line2: {
part1: before the link
part2: link text
part3: after the link text
link: href
ref: download reference
}

Reply:
Moved the links to nunjucks files and renamed references